django_evaluation/settings/local.py

```diff
@@ -281,6 +298,12 @@ def _set_favicon(html_color: str, project_root: Path) -> None:
     "variable",
 ]
 MENU_ENTRIES = []
+CHAT_BOT_URL = "http://vader5-icpub.lvt.dkrz.de:8502"
```

Comment:
This should be configurable via `os.environ.get`. @antarcticrainforest is this how you want this? I lost track when we are using `web_config.get` and when `os.environ.get`